src/App/global.js

```diff
@@ -0,0 +1,13 @@
+
+//This file is used to store all global variables
+
+
+//constant STUDENT_SESSION is used to enable/disable student applications to be filed for Student Sessions.
+//Value at index 0 is used to enable/disable the meny item which leads to the application screen.
+//Value at index 1 is used to enable/disable the submit button at the application screen.
+//Value at index 2 is used to display information appropriate for enabled/disabled student sessions at the student home screen.
+//Value at index 3 is used to display information appropriate for enabled/disabled student sessions at student session home screen.
+
+export const STUDENT_SESSION = [false, false, false, false];
```

Comment:
To make code referencing globals easier to read without opening `globals.js`, consider using separate constants instead.

Example
```js
export const STUDENT_SESSION_MENU_VISIBLE = false;
export const STUDENT_SESSION_SUBMIT_ENABLE = false;
// ...
```

This way webpack can give you compile time assurance that your globals are defined as well.

Also, are three different visibility modifiers really needed? Shouldn't it be hidden everywhere if it is disabled?